phpBB/phpbb/cache/driver/file.php

```diff
@@ -584,4 +587,19 @@ function _write($filename, $data = null, $expires = 0, $query = '')
 
 		return $return_value;
 	}
+
+	/**
+	* Change the cache path name
+	*
+	* @param string $name filename
+	* @return string the corrected path
+	*/
+	function get_file_name($name)
+	{
+		// remove slashes and replace with -
+		// as we can't write to a non-existent directory
+		$name = str_replace('/', '-', $name);
+
+		return $name;
```

Comment:
Just do `return str_replace('/', '-', $name);` instead of re-assigning the str_replace output above.